modules/doc/content/help/inl/hpc_binary.md

```diff
@@ -0,0 +1,251 @@
+assumed that BISON is the desired application. Others include "GRIFFIN" and "BLUE_CRAB". The
+correct environment is loaded using the module command as follows.
+
+```bash
+module load use.moose moose-apps BISON
+```
+
+At this point you should be able to run the executable. For this example, the BISON executable
+help can be displayed by running the following command.
+
+```bash
+/apps/herd/bison/bison-opt --help
+```
+
+## Create a Test Input File
+
+To demonstrate the use of the binary and to verify that it is operating correctly, create a
+test input file. For example, the following input file models steady-state diffusion and
+works with any MOOSE-based application.
+
+!listing moose/test/tests/kernels/simple_diffusion/simple_diffusion.i
+
+This file can be created in your INL-HPC home directory or copied via scp from your local machine.
+In both cases, first create a location for the file. For example, the following creates
+a "testing" folder in your remote home directory.
+
+```bash
+mkdir ~/testing
+```
+
+If you want to create the file on the remote machine you will need to use a terminal editor to create
+the file such as "emacs" or "vi". Alternatively, you can create the file on a local machine and copy
+it to the remote, by running the following from your local machine, assuming you created a
+file "input.i" with the aforemention content.
+
+```bash  style=background-color:#151B54;
+scp /path/to/the/local/input.i <your hpc user id>@hpclogin:~/testing/input.i
+```
+
+## Running Test Simulation
+
+### (1) Start-up an interactive job
+
+As mentioned above simulations should not be executed on the login nodes. For this example
+start an interactive job by requesting a single processor for one hour within the "moose"
+project queue.
+
+```bash
+qsub -I -l select=1:ncpus=1 -l walltime=1:0:0 -P moose
+```
+
+This command will likely take a few seconds or even minutes to execute depending the activity
+on the machine. When it completes you should see a message similar to the following.
+
+```bash
+qsub: waiting for job 141541.sawtoothpbs to start
+qsub: job 141541.sawtoothpbs ready
+```
+
+Again, you can verify that you are on a compute node using the `echo $HOSTNAME` command, which
+should give you a node name. For example, on sawtooth an example node name is "r1i2n34".
+
+When running in interactive mode you will also need to load the environment again.
+
+```bash
+module load use.moose moose-apps BISON
+```
+
+### (2) Run the simulation
+
+In the previous section a test input file was created in your INL-HPC home directory within
+the "testing" folder. MOOSE-based applications are designed to be executed from the location
+of the input file, so start by navigating to this location.
+
+```bash
+cd ~/testing
+```
+
+Next, execute the application with the "-i" argument followed by the input filename. For
+example, when using BISON with the created test input file.
+
+```bash
+/apps/herd/bison/bison-opt -i input.i
+```
```

Comment:
It might be worth pointing out to users that they won't be able to navigate to this folder somewhere. They will get a permission denied error and some might give up at that point.